processing/data_collection/gazette/spiders/pr_maringa.py

```diff
@@ -0,0 +1,48 @@
+from dateparser import parse
+from datetime import date, datetime
+import re
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrMaringaSpider(BaseGazetteSpider):
+    TERRITORY_ID = '4115200'
+    name = 'pr_maringa'
+    allowed_domains = ['maringa.pr.gov.br']
+
+    def start_requests(self):
+        """
+        @url http://venus.maringa.pr.gov.br/arquivos/orgao_oficial/paginar_orgao_oficial_ano.php
+        @returns requests 1
+        """
+        todays_date = date.today()
+        current_year = todays_date.year
+        for year in range(current_year, 2006, -1):
+            yield scrapy.FormRequest(
+                'http://venus.maringa.pr.gov.br/arquivos/orgao_oficial/paginar_orgao_oficial_ano.php',
+                headers={
+                    'Content-Type': 'application/x-www-form-urlencoded'
+                },
+                formdata={
+                    'ano': str(year),
+                    'entrar': 'Buscar'
+                },
+                callback=self.parse_year
+            )
+
+    def parse_year(self, response):
+        rows = response.css('table tr')[1:]
+        for row in rows:
+            gazette_id = row.css('td:nth-child(1) a::attr(href)').re_first('.*/[oO]{2}[mM] (.*)\.pdf')
+            gazette_date = row.css('td:nth-child(2) font > font::text').extract_first()
+            yield Gazette(
+                date=parse(gazette_date).date(),
```

Comment:
Perhaps I've expressed myself badly here or ```, languages=['pt']``` was removed by accident.
My review was about the f-string, the ```, languages=['pt']``` should be maintained. Sorry for the trouble